guibot/target.py

```diff
@@ -512,42 +494,40 @@ class Chain(Target):
     step did not succeed.
     """
 
-    def __init__(self, target_name, match_settings=None):
+    def __init__(self, target_name: str, match_settings: "Finder" = None) -> None:
         """
         Build an chain object.
 
-        :param str target_name: name of the target for all steps
+        :param target_name: name of the target for all steps
         :param match_settings: predefined configuration for the CV backend if any
-        :type match_settings: :py:class:`finder.Finder` or None
         """
         super(Chain, self).__init__(match_settings)
         self.target_name = target_name
         self._steps = []
         self.load(self.target_name)
 
-    def __str__(self):
+    def __str__(self) -> str:
         """Provide the target name."""
         return self.target_name
 
-    def __iter__(self):
+    def __iter__(self) -> Any:
```

Comment:
We can be more specific and use `Target`

Reply:
Done

Reply:
And even better `Iterator[Target]`